src/components/TransactionStatusToast/StatusIcon.tsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import { faInfo, faTimes, faWarning } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import classNames from 'classnames';
-import globalStyles from 'assets/sass/main.scss';
+
 import { StatusIconType } from './transactionStatusToast.types';
+
+import globalStyles from 'assets/sass/main.scss';
 import styles from './transactionStatusToastStyles.scss';
 
-type StatusIconProps = {
+export interface StatusIconPropsType {
   type: StatusIconType;
-};
+}
 
-export const StatusIcon = ({ type }: StatusIconProps) => {
+export const StatusIcon = ({ type }: StatusIconPropsType) => {
```

Comment:
can use type inline because it's a single proprety

Reply:
Changed. 👍